src/__stories__/navigation-breadcrumbs-story.tsx

```diff
@@ -11,8 +12,8 @@ type Args = {
 
 export const Default: StoryComponent<Args> = ({inverse}) => {
     return (
-        <ThemeVariant isInverse={inverse}>
-            <div style={{background: inverse ? skinVars.colors.backgroundBrand : skinVars.colors.background}}>
+        <ResponsiveLayout isInverse={inverse} fullWidth>
```

Comment:
unrelated change. This story should use ResponsiveLayout instead of setting the background by hand (like all the other stories).